2/Dockerfile

```diff
@@ -33,6 +33,8 @@ EXPOSE 8080 50000
 RUN curl https://pkg.jenkins.io/redhat-stable/jenkins.repo -o /etc/yum.repos.d/jenkins.repo && \
     rpm --import https://pkg.jenkins.io/redhat-stable/jenkins-ci.org.key && \
     yum install -y centos-release-scl-rh && \
+    curl https://copr.fedorainfracloud.org/coprs/alsadi/dumb-init/repo/epel-7/alsadi-dumb-init-epel-7.repo -o /etc/yum.repos.d/alsadi-dumb-init-epel-7.repo && \
+    yum install -y dumb-init && \
```

Comment:
you can move dumb-init into INSTALL_PKGS

the yum install centos-release-scl-rh is actually installing a repository(I think), which is why it has to be done separately.